module/zcommon/zfs_fletcher.c

```diff
@@ -435,7 +436,7 @@ fletcher_4_impl_get(void)
 		break;
 	}
 
-	ASSERT3P(ops, !=, NULL);
+	VERIFY(ops);
```

Comment:
Let's leave this as an ASSERT3P.  There's no compelling reason to promote it to a VERIFY.